doc-content/enterprise-only/installation/clustering-smart-router-install-proc.adoc

```diff
@@ -71,16 +69,17 @@ KieServicesConfiguration config = KieServicesFactory.newRestConfiguration("http:
 +
 In this example, `smartrouter.example.com` is the Smart Router URL, and `USERNAME` and `PASSWORD` are the log in credentials for the Smart Router configuration.
 
-[NOTE]
-====
-You must create containers directly against the `kie-server`. For example:
-
+. To create a new container in a managed {KIE_SERVER} so that you can fill it with example data, send the following HTTP request:
```

Comment:
```suggestion
. To create a new container in an unmanaged {KIE_SERVER} so that you can fill it with example data, send the following HTTP request:
```